std/getopt.d

```diff
@@ -298,8 +298,7 @@ void main(string[] args)
     ))
 )
 
-$(B Options with multiple names)
-
+Options_with_multiple_names:
```

Comment:
Per Andrei's request. The usage of the colon section will make it very similar. Please see https://github.com/D-Programming-Language/phobos/pull/3347#discussion_r31442154